modules/cnf-upgrading-performance-addon-operator.adoc

```diff
@@ -1,135 +0,0 @@
-// Module included in the following assemblies:
-//
-// * scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
-
-:_content-type: PROCEDURE
-[id="upgrading-performance-addon-operator_{context}"]
-= Upgrading Performance Addon Operator
```

Comment:
@yanirq That's my question to you. The reason I ask is because of the last day2 TBD comment in the JIRA ticket itself. If they aren't using the Performance Addon Operator anymore, having the install and upgrade modules doesn't make sense. The only thing that might make sense to cover from my perspective is if they are upgrading a cluster that has the Performance Addon Operator installed. Does upgrade remove this automatically, or does the user need to do it?

Reply:
PAO is removed automatically